dist/betteR20-5etools.user.js

```diff
@@ -2797,6 +2823,7 @@ function baseTool () {
 								<input type="checkbox">
 								<span class="name readable">${it.path ? `${it.path} / ` : ""}${it.name}</span>
 								${it.archived ? "<span class=\"name readable\">(archived)</span>" : ""}
+								${it.table ? "<span class=\"name readable\">(table)</span>" : ""}
```

Comment:
prefer template strings over escaping quotes: `<span class="name readable">(table)</span>`
(and the "archived" one too, please)